doc/cephfs/fs-volumes.rst

```diff
@@ -260,6 +260,31 @@ List subvolumes using::
 
 .. note:: subvolumes that are removed but have snapshots retained, are also listed.
 
+Set custom metadata on the subvolume as a key-value pair using::
+
+    $ ceph fs subvolume metadata set <vol_name> <subvol_name> <key_name> <value> [--group_name <subvol_group_name>]
+
+.. note:: If the key_name already exists then the old value will get replaced by the new value.
+
+.. note:: key_name and value should be string of ASCII characters with arbitrary lenght. key_name is case-insensitive and always stored in lower case.
+
+.. note:: Custom metadata set on a subvolume is not preserved by subvolume snapshot clone.
```

Comment:
Custom metadata on a subvolume is not preserved when snapshotting the subvolume, and hence, also not preserved when cloning the subvolume snapshot.